recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ requirements:
     - numpy
     - pandas
     - pip
+    - prometheus-flask-exporter
```

Comment:
We added this new dependency with MLflow 1.5.0 - it's only available on conda via an out-of-date / seemingly unmaintained channel (https://anaconda.org/giperball/prometheus-flask-exporter). As described in https://github.com/conda/conda-build/issues/548, it's not currently possible to include PyPI-only dependencies in conda recipes, & there doesn't seem to have been progress on allowing recipes to depend on packages in custom conda channels (see https://github.com/conda/conda/issues/1504), so will remove the `prometheus-flask-exporter` dependency for now.

Reply:
These dependencies are straight-forward to add: https://github.com/conda-forge/staged-recipes/pull/10505 No need to remove them.

Reply:
Ah thanks @xhochy! Now that it's on conda-forge, would be happy to approve a PR adding the `prometheus-flask-exporter` dependency to the MLflow recipe